app/models/article.rb

```diff
@@ -112,9 +112,10 @@ class Article < ApplicationRecord
 
   scope :admin_published_with, lambda { |tag_name|
     published
-      .where(user_id: SiteConfig.staff_user_id)
-      .order(published_at: :desc)
-      .tagged_with(tag_name)
+      .where(user_id: User.with_role(:super_admin)
+                          .union(User.with_role(:admin))
+                          .union(id: SiteConfig.staff_user_id)
```

Comment:
Is this line included because the staff user is not necessarily someone with an admin role yet? If they are an admin out of the box, I imagine this line could be omitted?

Reply:
> Is this line included because the staff user is not necessarily someone with an admin role yet?

Correct. It was a much bigger change to make the staff user an admin since we would have to update the code and then we would have to run a data update script to update all Forems so I went with this solution instead.